composer.json

```diff
@@ -37,13 +38,18 @@
         }
     },
     "scripts": {
-        "lint": "parallel-lint src tests",
-        "phpcs": "phpcs src tests --standard=psr2 -sp --colors",
-        "phpunit": "phpunit --verbose --colors=always",
-        "test": [
-            "@lint",
-            "@phpunit",
-            "@phpcs"
-        ]
-    }
+       "lint": "parallel-lint src tests",
```

Comment:
The indentation for these lines appears off. Please make sure it fits the indentation of the rest of the file (4 spaces per indentation level). Thanks!

Reply:
done!